src/components/astro/AstroClassroomsTable.jsx

```diff
@@ -90,7 +90,7 @@ const AstroClassroomsTable = (props) => {
                 props.assignmentsStatus === ASSIGNMENTS_STATUS.SUCCESS) &&
                 props.assignments[classroom.id].map((assignment) => {
                   const projectUrl = (assignmentsMetadata && assignmentsMetadata[assignment.workflowId]) ?
-                    `${config.zooniverse}/projects/${assignmentsMetadata[assignment.workflowId].slug}/classify?group=${classroom.zooniverse_user_group_id}` :
+                    `${config.zooniverse}/projects/${assignmentsMetadata[assignment.workflowId].slug}/classify?group=${classroom.attributes.zooniverse_group_id}` :
```

Comment:
Do we need an existential check for `classroom.attributes` or can we be guaranteed there'll at least be some sort of object there?